### PR TITLE
Gruntfile.js - add task for removing old static release files

### PR DESCRIPTION
**Pull request by MariagraziaAlastra**
So, I added a gitrm task that takes care of removing all the old release files, a gitadd task that adds the newly released files, and an exec:commit_static task that commits all the new release files, plus package.json.
Can't use gitcommit anymore because it doesn't see deleted files.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -190,29 +222,6 @@ module.exports = function(grunt) {
...
-                    message: 'Release IA pages version: <%= pkg.version %>'
-                },
-                files: {
-                    src: [
-                        static_dir + 'js/ia0.*.0.js',
-                        static_dir + 'js/ddgc0.*.0.js',
-                        static_dir + 'css/ddgc0.*.0.css',
-                        static_dir + 'css/ia0.*.0.css',
-                        static_dir + 'css/ddgc.css',
-                        'package.json'
```
Comment (jbarrett) [inline Gruntfile.js]: Is there a reason we're leaving out package.json now?

Comment (MariagraziaAlastra) [inline Gruntfile.js]: I'm not using gitcommit anymore, I'm using an exec task instead: https://github.com/duckduckgo/community-platform/pull/1083/files#diff-35b4a816e0441e6a375cd925af50752cR270
As you can see package.json is included there

Comment (jbarrett) [inline Gruntfile.js]: Missed that, thanks!

File: Gruntfile.js [hunk 2]
```diff
@@ -176,6 +178,37 @@ module.exports = function(grunt) {
...
+        gitadd: {
+            options: {
+                force: 'true'
+            },
+            files: {
+                src: [
+                    static_dir + 'js/ia0.*.0.js',
+                    static_dir + 'js/ddgc0.*.0.js',
+                    static_dir + 'css/ddgc0.*.0.css',
+                    static_dir + 'css/ia0.*.0.css'
```
Comment (MariagraziaAlastra) [inline Gruntfile.js]: @jbarrett Need a comma here

Comment (jbarrett) [inline Gruntfile.js]: Oops, thanks!

**Comment by MariagraziaAlastra** [comment]
@jbarrett Looks good! :+1:

**Comment by jbarrett** [comment]
Thanks @MariagraziaAlastra - sorry for mix up.

**Pull request merged**